integration-tests/bridges/run-test.sh

```diff
@@ -7,21 +7,29 @@ trap 'kill -9 -$$ || echo "Environment already teared down"' SIGINT SIGTERM EXIT
 test=$1
 
 export LOCAL_BRIDGE_TESTING_PATH=~/local_bridge_testing
-export DOWNLOADS_PATH=$LOCAL_BRIDGE_TESTING_PATH/downloads
-mkdir -p $DOWNLOADS_PATH
-
-# Download the bridge testing "framework" from the `polkadot-sdk` repo
-# to `~/local_bridge_testing/downloads/polkadot-sdk`.
-framework_repo_path=$DOWNLOADS_PATH/polkadot-sdk
-rm -rf $framework_repo_path
-git clone --branch master -n --depth=1 --filter=tree:0 \
-  https://github.com/paritytech/polkadot-sdk.git $framework_repo_path
-pushd $framework_repo_path
-git sparse-checkout set --no-cone bridges/testing/framework
-git fetch --tags
-git checkout polkadot-v1.11.0
-popd
+
+if [ -z "$FRAMEWORK_REPO_PATH" ]; then
+  # Download the bridge testing "framework" from the `polkadot-sdk` repo
+  # to `~/local_bridge_testing/downloads/polkadot-sdk`.
+  export DOWNLOADS_PATH=$LOCAL_BRIDGE_TESTING_PATH/downloads
+  echo "FRAMEWORK_REPO_PATH is NOT set, so downloading 'polkadot-sdk' repo to the: $DOWNLOADS_PATH"
+  exit 1
```

Comment:
Does it make sense to `exit` immediately after informing the user the PSDK will be downloaded, or is this just a breakpoint that was forgotten to be removed?

Reply:
:see_no_evil: uff, thank you, yes breakpoint